src/release/main.d

```diff
@@ -183,6 +183,20 @@ void main ( string[] params )
         }
     }
 
+    auto email_body = craftMail(con, repo, myrelease.type,
+        list.actions
+            .map!(a=>cast(ReleaseAction) a)
+            .filter!(a=>a !is null)
+            .array);
+
+    writefln("This is the announcement email:\n-----\n%s\n-----", email_body);
+
+    if (readYesNoResponse("Would you like to send it to <%s> now?",
+        getConfig("neptune.mail-recipient")))
+    {
+        sendMail(email_body);
```

Comment:
Pass the recipients here instead of querying the config again inside? I have the feeling it will be harder to maintain consistency as the code evolves this way (and higher risk of getting a typo in the string, which is not *type-checked*).